prow/config/config.yaml

```diff
@@ -84,8 +84,13 @@ branch-protection:
           # Use this to specify that a status coming from outside of prow is
           # required.  We use this to require functional jobs running in
           # jenkins are required, for example.
-          required_status_checks:
-            contexts: ["test-v1b1-integration"]
+          branches:
+            main:
+              required_status_checks:
+                contexts: ["test-v1b1-integration"]
+            master:
+              required_status_checks:
+                contexts: ["test-v1b1-integration"]
```

Comment:
Do  we need this still, I expect we will point all open PRs in BMO to main once we switch to it?

Reply:
This has to be removed once main branch is default and all the PRs are pointing to main. In the meanwhile this is needed.